### PR TITLE
Rename and reorder `data.ts`

### PR DESCRIPTION
**Pull request by Skarvion**
Minor tweaks to some countries to follow ISO 3166 closer.

Not all countries are changed though because some listed in ISO 3166 uses non ASCII characters such as Turkey. So, I kept them as it is

### CODE REVIEW

File: src/data.ts [hunk 1]
```diff
@@ -92,6 +92,7 @@ const countries: CountryInterface[] = [
   { name: 'Eritrea', dial_code: '+291', code: 'ER', flag: '🇪🇷' },
   { name: 'Estonia', dial_code: '+372', code: 'EE', flag: '🇪🇪' },
   { name: 'Ethiopia', dial_code: '+251', code: 'ET', flag: '🇪🇹', local_name: 'ኢትዮጵያ' },
+  { name: 'Eswatini', dial_code: '+268', code: 'SZ', flag: '🇸🇿' },
```
Comment (necessarylion) [inline src/data.ts:95]: Could you reorder this please?
I think `Eswatini` should come before `Ethiopia`

File: src/data.ts [hunk 2]
```diff
@@ -226,12 +226,13 @@ const countries: CountryInterface[] = [
...
   { name: 'Niue', dial_code: '+683', code: 'NU', flag: '🇳🇺' },
   { name: 'Norfolk Island', dial_code: '+672', code: 'NF', flag: '🇳🇫', part_of: ['AU'] },
+  { name: 'North Macedonia', dial_code: '+389', code: 'MK', flag: '🇲🇰' },
   { name: 'Northern Mariana Islands', dial_code: '+1670', code: 'MP', flag: '🇲🇵', part_of: ['US'] },
   { name: 'Norway', dial_code: '+47', code: 'NO', flag: '🇳🇴', local_name: 'Norge' },
   { name: 'Oman', dial_code: '+968', code: 'OM', flag: '🇴🇲', local_name: 'عُمان' },
   { name: 'Pakistan', dial_code: '+92', code: 'PK', flag: '🇵🇰', local_name: 'پاکستان' },
   { name: 'Palau', dial_code: '+680', code: 'PW', flag: '🇵🇼' },
-  { name: 'Palestinian Territory, Occupied', dial_code: '+970', code: 'PS', flag: '🇵🇸' },
+  { name: 'Palestinian, State of', dial_code: '+970', code: 'PS', flag: '🇵🇸' },
```
Comment (necessarylion) [inline src/data.ts:235]: `Palestine` ?

**Review by necessarylion — CHANGES_REQUESTED** [review]
Thanks for the PR.

**Comment by Skarvion** [comment]
Silly me, fixed the code

**Review by necessarylion — APPROVED** [review]

**Pull request merged**